cookbooks/omnitruck/files/poller-cron.sh

```diff
@@ -1,3 +1,14 @@
 #!/bin/bash
 
+PATH=$(hab pkg path core/bundler)/bin:$(hab pkg path core/ruby)/bin:$(hab pkg path
+core/coreutils)/bin:/usr/local/bin:/usr/local/sbin/:/usr/bin:/usr/sbin/:/bin:/sbin
+RUBY_VERSION="$(hab pkg path core/ruby | rev | cut -d '/' -f 2 | rev | cut -d '.' -f 1-2).0"
```

Comment:
I like your crazy style here--reminds me of this: https://github.com/habitat-sh/habitat/blob/7fd4ffc05d4e612acf0396420d112986eb099662/components/studio/build-docker-image.sh#L84